src/Groups/cosets.jl

```diff
@@ -65,13 +74,72 @@ function show(io::IO, x::GroupCoset)
    a = GAP.gap_to_julia(GAP.Globals.StringView(x.H.X))
    b = GAP.gap_to_julia(GAP.Globals.StringView(x.repr.X))
    if x.side == :right
-      print(io, a, " * ", b)
+      print(io, "Right coset   ", a, " * ", b)
    else
-      print(io, b, " * ", a)
+      print(io, "Left coset   ", b, " * ", a)
    end
    return nothing
 end
 
+"""
+    isleftcoset(c::GroupCoset)
+Return whether the coset `c` is a left coset of its acting domain.
+"""
+isleftcoset(c::GroupCoset) = c.side == :left
+
+"""
+    isrightcoset(c::GroupCoset)
+Return whether the coset `c` is a right coset of its acting domain.
+"""
+isrightcoset(c::GroupCoset) = c.side == :right
+
+"""
+    as_right_coset(c::GroupCoset)
+Return a coset (either left or right) as a right coset. 
+"""
+function as_right_coset(c::GroupCoset)
+   if c.side == :right return c
+   else return right_coset(c.H^(c.repr^-1), c.repr)
+   end
+end
```

Comment:
To repeat the comment I made previously:

I don't understand the purpose of this function. What would it be used for? And from the description, it is completely what it should be mean to "return a left coset as a right coset" (of course I can look into your code and see what it does, that's not the point).

In general I have doubts that this is useful. Do you have any particular application in mind?

Reply:
If `lc` = `xH` is a left coset, then `as_right_coset(lc)` returns a right coset `Kx` (the representative is the same) such that `xH` = `Kx` as sets. At the moment I don't have in mind any particular use of this function. Is it the case to delete it?

Reply:
That doesn't seem like a very useful operation (and if a user wants it, it is trivial to implement). So yeah, I'd delete it, and only add it back if we have an actual use case.

Reply:
Ok, I'll delete it.

Reply:
Please delete them, then ping me, then I will review one last time and merge.